src/applications/disability-benefits/all-claims/tests/config/submitForm.unit.spec.js

```diff
@@ -20,6 +25,10 @@ describe('Form 526 submit reject timer', () => {
     requests = [];
   });
 
+  after(() => {
+    localStorage.getItem.restore();
```

Comment:
Could potentially actually test that the CSRF token was fetched. Something like:

```
expect(stubbedGetItem.calledWith('csrfToken')).to.be.true;
```